app/views/projects.html

```diff
@@ -17,13 +20,10 @@ <h2 class="text-center">Loading...</h2>
                   <h1>My Projects</h1>
                   <div class="projects-options">
                     <div class="projects-add" ng-if="canCreate">
-                      <button ng-click="createProject()" class="btn btn-md btn-primary">
+                      <button ng-click="createProject($event)" class="btn btn-md btn-primary">
```

Comment:
Same comment on `btn-md`, although I see that was already there.

Reply:
removed the btn-md